Gopkg.toml

```diff
@@ -0,0 +1,42 @@
+# Gopkg.toml example
```

Comment:
I'm good to switch `glide` to `dep`. Can you

1) remove glide.yaml/glide.lock
2) add Gopkg.lock also

Reply:
3. change .travis.yml to install `dep` rather than `glide`
4. modify `deps` section in Makefile to be `dep xxxx` to install dependent go packges.

Reply:
I honestly didn't notice that there was glide.